<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="4" hostname="IND03687" name="src.com.salesforce.SingleRun.MVRTest_SingleRun_ClmRteDevEnv" tests="5" failures="0" timestamp="17 Feb 2020 10:13:13 GMT" time="22.958" errors="1">
  <testcase name="sendEmailTest" time="0.000" classname="src.com.salesforce.SingleRun.MVRTest_SingleRun_ClmRteDevEnv">
    <skipped/>
  </testcase> <!-- sendEmailTest -->
  <testcase name="clmPresentationMetadataReport" time="0.000" classname="src.com.salesforce.SingleRun.MVRTest_SingleRun_ClmRteDevEnv">
    <skipped/>
  </testcase> <!-- clmPresentationMetadataReport -->
  <testcase name="keyMessageMetadataReport" time="0.000" classname="src.com.salesforce.SingleRun.MVRTest_SingleRun_ClmRteDevEnv">
    <skipped/>
  </testcase> <!-- keyMessageMetadataReport -->
  <testcase name="login" time="21.620" classname="src.com.salesforce.SingleRun.MVRTest_SingleRun_ClmRteDevEnv">
    <skipped/>
  </testcase> <!-- login -->
  <testcase name="logout" time="1.338" classname="src.com.salesforce.SingleRun.MVRTest_SingleRun_ClmRteDevEnv">
    <error type="java.lang.ExceptionInInitializerError">
      <![CDATA[java.lang.ExceptionInInitializerError
	at src.com.salesforce.SingleRun.MVRTest_SingleRun_ClmRteDevEnv.logout(MVRTest_SingleRun_ClmRteDevEnv.java:1092)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.TestRunner.afterRun(TestRunner.java:958)
	at org.testng.TestRunner.run(TestRunner.java:606)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: java.lang.IllegalStateException: The path to the driver executable must be set by the webdriver.chrome.driver system property; for more information, see https://github.com/SeleniumHQ/selenium/wiki/ChromeDriver. The latest version can be downloaded from http://chromedriver.storage.googleapis.com/index.html
	at com.google.common.base.Preconditions.checkState(Preconditions.java:847)
	at org.openqa.selenium.remote.service.DriverService.findExecutable(DriverService.java:125)
	at org.openqa.selenium.chrome.ChromeDriverService.access$000(ChromeDriverService.java:35)
	at org.openqa.selenium.chrome.ChromeDriverService$Builder.findDefaultExecutable(ChromeDriverService.java:156)
	at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:346)
	at org.openqa.selenium.chrome.ChromeDriverService.createDefaultService(ChromeDriverService.java:91)
	at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:123)
	at com.salesforce.genericLib.Driver.<clinit>(Driver.java:13)
	... 24 more
]]>
    </error>
  </testcase> <!-- logout -->
</testsuite> <!-- src.com.salesforce.SingleRun.MVRTest_SingleRun_ClmRteDevEnv -->
